scripts/mutect2_wdl/README.md

```diff
@@ -162,16 +167,18 @@ gs://broad-dsde-methods/takuto/na12878-crsp-ice/SM-612V3.bam    gs://broad-dsde-
   "Mutect2_Multi.pon": "/data/m1/refseq_exome_10bp_hg19_300_1kg_normal_panel.vcf",
   "Mutect2_Multi.pon_index": "/data/m1/refseq_exome_10bp_hg19_300_1kg_normal_panel.vcf.idx",
   "Mutect2_Multi.scatter_count": 2,
-  "Mutect2_Multi.dbsnp": "/data/m1/dbsnp_134_b37.leftAligned.vcf",
-  "Mutect2_Multi.dbsnp_index": "/data/m1/dbsnp_134_b37.leftAligned.vcf.idx",
-  "Mutect2_Multi.cosmic": "/data/m1/hg19_cosmic_v54_120711.vcf",
-  "Mutect2_Multi.cosmic_index": "/data/m1/hg19_cosmic_v54_120711.vcf.idx",
+  "Mutect2_Multi.gnomad": "/data/m2/gnomad.vcf",
+  "Mutect2_Multi.gnomad_index": "/data/m2/gnomad.vcf.idx",
+  "Mutect2_Multi.variants_for_contamination": "/data/m2/gnomad-common-biallelic-snps.vcf",
+  "Mutect2_Multi.variants_for_contamination_index": "/data/m2/gnomad-common-biallelic-snps.vcf.idx",
   "Mutect2_Multi.is_run_orientation_bias_filter": true,
   "Mutect2_Multi.is_run_oncotator": true,
   "Mutect2_Multi.m2_docker": "broadinstitute/gatk:1.0.0.0-alpha1.2.4",
   "Mutect2_Multi.oncotator_docker": "broadinstitute/oncotator:1.9.2.0",
   "Mutect2_Multi.preemptible_attempts": 2,
-  "Mutect2_Multi.onco_ds_tar_gz": "/data/onco_dir/oncotator_v1_ds_April052016.tar.gz"
+  "Mutect2_Multi.onco_ds_tar_gz": "/data/onco_dir/oncotator_v1_ds_April052016.tar.gz",
+  "Mutect2_Multi.m2_extra_args": "--maxNumHaplotypesInPopulation 50 --tumor_lod_to_emit 4.0",
```

Comment:
Not standard at all, but I chose them to be not preposterous.  You might want to reduce haplotypes for speed, and you might want to raise `tumor_lod_to_emit` to trim the output vcf.